docker/pravega/entrypoint.sh

```diff
@@ -26,12 +26,36 @@ configure_controller() {
     echo "JAVA_OPTS=${JAVA_OPTS}"
 }
 
-configure_segmentstore() {
-    add_system_property "pravegaservice.zkURL" "${ZK_URL}"
-    add_system_property "autoScale.controllerUri" "${CONTROLLER_URL}"
+configure_nfs_s3_hdfs() {
```

Comment:
I'm not sure why we are hardcoding `nfs_s3_hdfs` in the function name, shouldn't it be `configure_tiertwo` to make it generic so that we can change it arbitrarily in the future?